avaframe/out3Plot/outCom1DFA.py

```diff
@@ -279,3 +285,90 @@ def addResult2Plot(ax, header, rasterData, resType, colorbar=True, contour=False
     else:
         CS = None
     return ax, extent, cb, CS
+
+
+def createContourPlot(reportDictList, avalancheDir, simDF):
+    """ create a contour line plot of all simulations of current run
+        option: in x, y coordinate system or in aimec s,l coordinate system
+        s, l only works if createRangeTimeDiagram=True and tt-diagram = True
+
+        Parameters
+        -----------
+        reportDictList: list
+            list of com1DFA dictionary with info on contour dicts for each sim
+        avalancheDir: str or pathlib path
+            path to avalanche directory
+
+        Returns
+        -------
+        reportDictList: list
+            updated reportDictList - deleted contours dict
+    """
+
+    modName = 'com1DFA'
+    contourD = {}
+    contD = simDF['contourType'].iloc[0]
+    # check if consistent settings throughout all sims
+    if len(np.unique(simDF['contourType'])) != 1:
+        log.warning('Contour plot type is not identical for all sims performed - so cannot create contour plot')
+        return reportDictList, False
+
+    # fetch coordinates of contour line for each sim in dict
+    if contD in reportDictList[0]:
+        for cont in reportDictList:
+            contourD[list(cont[contD].keys())[0]] = cont[contD][list(cont[contD].keys())[0]]
+            del cont[contD]
```

Comment:
a new contourDict is created (read from reporDictList) and contours key is deleted in reportDicts in reportDictList so that the report generation works fine